sites/fast-website/src/app/components/feature-card/feature-card.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from "@microsoft/fast-element";
+import { display, neutralForegroundRestBehavior } from "@microsoft/fast-components";
+
+export const FeatureCardStyles = css`
+    ${display("grid")} :host {
+        contain: layout;
+        grid-template-columns: 1fr 2fr;
+        color: var(--neutral-foreground-rest);
+        font-family: var(--text-font);
+        box-sizing: border-box;
+        padding: 20px;
+        box-shadow: unset;
+        height: 140px;
+        width: 600px;
+    }
+
+    :host::before {
+        content: "";
+        display: block;
+        background-color: currentColor;
+        position: fixed;
+        height: 1px;
+        width: 93%;
+        left: 20px;
+        top: -1px;
+    }
+
+    :host(:hover) ::slotted(fast-anchor) {
+        opacity: 1;
+    }
+
+    :host(:hover)::before {
+        opacity: 0;
+    }
+
+    :host(:hover) ::slotted(h5) {
```

Comment:
Is this always going to be an h5?

Reply:
Not necessarily I suppose.  I could try an alternative selector to try and grab that element.

Reply:
Changed the selector from h5 to :first-child.